<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestWindow</class>
 <widget class="QMainWindow" name="RequestWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DataViz</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="MainGrid">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="TableSelector">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Auteur</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Publication</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Venue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mot Clé</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Chercher par :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="RButtonIdSelect">
              <property name="text">
               <string>Id</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="RButtonNameSelect">
              <property name="text">
               <string>Nom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="SingleSelectorInput"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_autor">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="AutorCheck">
                <property name="text">
                 <string>Co-auteur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="PublicationCheck">
                <property name="text">
                 <string>Publication</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="KeywordCheck">
                <property name="text">
                 <string>Keyword</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="VenueCheck">
                <property name="text">
                 <string>Venue</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_publication">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QCheckBox" name="AutorCheck_2">
                <property name="text">
                 <string>Auteur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="PublicationCheck_2">
                <property name="text">
                 <string>Publication</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="KeywordCheck_2">
                <property name="text">
                 <string>Keyword</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="VenueCheck_2">
                <property name="text">
                 <string>Venue</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Catégorie : </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="CategorySelector">
                  <item>
                   <property name="text">
                    <string>Aucune</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Article</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Collection</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Livre</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <item>
                 <widget class="QCheckBox" name="YearCheck">
                  <property name="text">
                   <string>Année</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>12</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>de</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="MinYearSelector">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimum">
                   <number>1900</number>
                  </property>
                  <property name="maximum">
                   <number>2020</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>8</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>à</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="MaxYearSelector">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimum">
                   <number>1900</number>
                  </property>
                  <property name="maximum">
                   <number>2020</number>
                  </property>
                  <property name="value">
                   <number>2020</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_venue">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="AutorCheck_3">
                <property name="text">
                 <string>Auteur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="PublicationCheck_3">
                <property name="text">
                 <string>Publication</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="KeywordCheck_3">
                <property name="text">
                 <string>Keyword</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_keywords">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QCheckBox" name="AutorCheck_4">
                <property name="text">
                 <string>Auteur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="PublicationCheck_4">
                <property name="text">
                 <string>Publication</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="VenueCheck_4">
                <property name="text">
                 <string>Venue</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Top : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="TopTypeSelector">
              <item>
               <property name="text">
                <string>Aucun</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>+</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>-</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QCheckBox" name="LimitResultCheck">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Max nb résultats : </string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="MaxResultSelector">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(171, 171, 171);</string>
              </property>
              <property name="text">
               <string>max : 500</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>100</number>
            </property>
            <property name="leftMargin">
             <number>100</number>
            </property>
            <property name="rightMargin">
             <number>100</number>
            </property>
            <item>
             <widget class="QPushButton" name="PreviewButton">
              <property name="text">
               <string>Prévisualiser La Requête</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ShowGraphButton">
              <property name="text">
               <string>Afficher Le Graph</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="PreviewTable"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LimitResultCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MaxResultSelector</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YearCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MinYearSelector</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YearCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MaxYearSelector</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TableSelector</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
